gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/PublisherSettings.java

```diff
@@ -123,17 +122,17 @@
         RetryParams.newBuilder()
             .setRetryBackoff(
                 BackoffParams.newBuilder()
-                    .setInitialDelayMillis(100L)
+                    .setInitialDelay(Duration.millis(100L))
                     .setDelayMultiplier(1.2)
-                    .setMaxDelayMillis(1000L)
+                    .setMaxDelay(Duration.millis(1000L))
                     .build())
             .setTimeoutBackoff(
                 BackoffParams.newBuilder()
-                    .setInitialDelayMillis(300L)
-                    .setDelayMultiplier(1.3)
-                    .setMaxDelayMillis(3000L)
+                    .setInitialDelay(Duration.millis(2000L))
```

Comment:
Yes, we have maximum and initial value for timeout. The initial timeout is 200L in first trial and then be multiplied by 1.5 in the second trial and so on. We use the |BackoffParams| for such settings for both retry and timeout.
I understand where the confusion comes from and I agree that we may need to reconsider the method names to fit both cases.  https://github.com/GoogleCloudPlatform/gcloud-java/issues/814